source/features/sticky-sidebar.css

```diff
@@ -21,6 +22,11 @@ Exclusively use simple sums and use `0px` instead of `0`
 	--rgh-sticky-notifitication-header-height: 69px;
 }
 
+/* required to make `sticky` work for the issue Sidebar */
+div:has(> div[data-testid='issue-viewer-metadata-pane']) {
+	display: unset;
+}
```

Comment:
I think it's better to set this via JS in trackSidebar. The `:has()` is heavy when its root contains a lot of elements, like in this case the generic `div`

Reply:
since the `display: unset` doesn't seem to affect the other sidebars
I changed it so a class gets set on all the parent objects and I then just use that class.